### ISSUE TITLE
Remove the dependency on the VSPackage Builder extension (VS2010)

### ISSUE THREAD

**Issue by EWSoftware** [EWSoftware/SHFB#11]
It's not available for VS2013 and going forward I'll need to get by without it.  Move the code files in the GeneratedCode\ folder to the root and merge the code into the derived classes where possible.  Button bitmap resources images can move to the Resources\ folder.

**Comment by sharwell** [comment]
I can implement this one and send a pull request if you want.

**Comment by EWSoftware** [comment]
I've already got the changes figure out.  I had to do the same thing for the spell checker project and there isn't much to do.

I see that you have a version of MPFProj here on GItHub deployed as a NuGet package.  It would make sense to move toward using that.  However, when I had a quick look at the code a while ago you were missing a few bug fixes that I had in the SHFB copy (one in Utilities.cs related to recursive copies and one in FileNode.cs related to in-place renaming of a file that caused a problem with source control are the two I recall).  If you'd like to review my version and make sure you've got the same set of changes, that would help.  I tagged all of my changes with "!EFW" in the code and commented the changes at the top of the files so they should be easy enough to find.  Once they match feature and bug fix-wise, it could be removed in favor of using the NuGet package.
